src/api/wiki.js

```diff
@@ -221,6 +277,8 @@ async function tree(path, depth, self, reverse, ordered) {
             if (aa[x] !== bb[x]) {
                 var c = Number(aa[x]),
                     d = Number(bb[x]);
+
+                // @ts-ignore: Loose comparison of number and string
                 if (c == aa[x] && d == bb[x]) {
```

Comment:
See https://github.com/mdn/kumascript/pull/1100#pullrequestreview-217042587 and https://github.com/Microsoft/TypeScript/issues/30655